Jenkinsfile_CNP

```diff
@@ -76,7 +76,9 @@ withPipeline(type, product, component) {
 
   enableDockerBuild()
   enableAksStagingDeployment()
-  disableLegacyDeployment()
+  onMaster() {
+    disableLegacyDeployment()
+  }
```

Comment:
@timja Is this safe enough?

Reply:
there could be any other branch build/pr raised